resources/views/backend/admin/create.blade.php

```diff
@@ -0,0 +1,82 @@
+@extends('backend.layouts.master')
+@section('title', trans('labels.admin'))
+@section('content')
+<!-- page content -->
+
+<div class="col-md-12 col-sm-12 col-xs-12">
+	<div class="x_panel">
+		<div class="x_title">
+			<h2>{!! trans('labels.admin') !!} <small>{!! trans('labels.create') !!}</small></h2>
+			<div class="clearfix"></div>
+		</div>
+		<div class="x_content">
+			<br />
+			{!! Form::open(['route' => 'admin.admins.store', 'method' => 'POST', 'files' => true, 'class' => 'form-horizontal form-label-left', 'id' => 'my-form']) !!}
```

Comment:
`'id' => 'my-form'` this id name is really not good, please use another name and it should be readable. Please remember naming is very importance.